espanol/jordiHurtado/inmortal/inmortal.md

```diff
@@ -1 +1,5 @@
-Y Jordi Hurtado creò una pociòn de inmmortalidad que terminò bebièndose èl.
\ No newline at end of file
+Y Jordi Hurtado creò una pociòn de inmmortalidad que terminò bebièndose èl.
+
+Tras dar buena cuenta de todo aquel ejército primero gracias a su inmortalidad, y tras otras múltples gestas y conquistas, Jordi estaba terriblemente aburrido.
+¡Qué hacer con tantos siglos y milenios como tenía por delante!
+[tuvo una idea brillante que cambiaría la parrilla de programación por toda la eternidad](saberYganar/saberYganar.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.